src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1394 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+from copy import deepcopy
+
+# Import Pyomo libraries
+from pyomo.environ import (
+    Set,
+    Var,
+    Param,
+    Suffix,
+    Constraint,
+    check_optimal_termination,
+    units as pyunits,
+)
+from pyomo.common.config import ConfigBlock, ConfigValue, In
+
+# Import IDAES cores
+from idaes.core import (
+    FlowsheetBlock,
+    ControlVolume0DBlock,
+    declare_process_block_class,
+    UnitModelBlockData,
+    useDefault,
+)
+from idaes.core.util.config import is_physical_parameter_block
+import idaes.core.util.scaling as iscale
+from idaes.core.solvers import get_solver
+from idaes.core.util.exceptions import ConfigurationError, InitializationError
+import idaes.logger as idaeslog
+
+# Import Watertap packages
+from watertap.property_models.seawater_prop_pack import SeawaterParameterBlock
+from watertap.property_models.water_prop_pack import WaterParameterBlock
+
+_log = idaeslog.getLogger(__name__)
+__author__ = "Zhuoran Zhang"
+
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.scaling import (
+    calculate_scaling_factors,
+    unscaled_variables_generator,
+    badly_scaled_var_generator,
+)
+
+
+@declare_process_block_class("MEDTVCSurrogate")
+class MEDTVCData(UnitModelBlockData):
+    """
+    Multi-effect distillation with thermal vapor compressor model
+    """
+
+    CONFIG = ConfigBlock()
+
+    CONFIG.declare(
+        "dynamic",
+        ConfigValue(
+            domain=In([False]),
+            default=False,
+            description="Dynamic model flag - must be False",
+            doc="""Indicates whether this model will be dynamic or not,
+    **default** = False. """,
+        ),
+    )
+    CONFIG.declare(
+        "has_holdup",
+        ConfigValue(
+            default=False,
+            domain=In([False]),
+            description="Holdup construction flag - must be False",
+            doc="""Indicates whether holdup terms should be constructed or not.
+    **default** - False.""",
+        ),
+    )
+    CONFIG.declare(
+        "property_package_water",
```

Comment:
```suggestion
        "property_package_liquid",
```

Reply:
@adam-a-a Should also make these changes in LT-MED for consistency.

Reply:
I think it should be a separate PR for LT-MED consistent.

Reply:
This renaming is applied to all scripts, and corresponding changes in LT-MED will be made in a separate PR.